Docs/install/install_docker.rst

```diff
@@ -37,132 +37,255 @@
 .. _installation-docker:
 
 ##############################
-AIMET Installation in Docker
+AIMET Docker install and setup
 ##############################
 
-This page provides instructions to install AIMET package inside a development docker container.
+This page describes how to install AIMET in a development Docker container.
 
-Set variant
-~~~~~~~~~~~
-Set the `<variant_string>` to ONE of the following depending on your desired variant
-    #. For the PyTorch 2.1 GPU variant, use `torch-gpu`
-    #. For the PyTorch 2.1 CPU variant, use `torch-cpu`
-    #. For the PyTorch 1.13 GPU variant, use `torch-gpu-pt113`
-    #. For the TensorFlow GPU variant, use `tf-gpu`
-    #. For the TensorFlow CPU variant, use `tf-cpu`
-    #. For the ONNX GPU variant, use `onnx-gpu`
-    #. For the ONNX CPU variant, use `onnx-cpu`
+Prerequisites
+=============
+
+Ensure that you have Docker installed.
+
+
+Installing AIMET
+================
+
+To install an AIMET development Docker container, you:
+
+1. Decide on an AIMET variant to install
+2. Choose to either:
+   1. Download a prebuilt Docker image
```

Comment:
The rendering is showing up on a single line like this: Can we have separate lines and then use 'a' and 'b' for the sub-bullets?
`Choose to either: 1. Download a prebuilt Docker image 2. Build a Docker image
`

Reply:
Formatting fixed.
Changed to bullets rather than numbers.